### PR TITLE
Enhance default config (#2005)

### PR DESCRIPTION
**Pull request by povsister**
Make it faster&easier to deploy LP without annoying WARNINGS when using external DB storage.

### CODE REVIEW

File: bukkit/src/main/resources/config.yml [hunk 1]
```diff
@@ -136,6 +136,15 @@ data:
     properties:
       useUnicode: true
       characterEncoding: utf8
+      # If you are running local Database instance(For example: MySQL) on the same server of minecraft.
+      # You may have the following two options uncommented.(Remove the leading "#")
+      # Since there is no need to encrypt DB connection.
+      # It helps you get rid of annoying "SSL certificate ERROR" spamming your console.
+      # More properties definitions for MySQL can be found at
+      # https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html
+      #####
```
Comment (Andre601) [inline bukkit/src/main/resources/config.yml]:
```suggestion
      #
```

File: bungee/src/main/resources/config.yml [hunk 2]
```diff
@@ -133,6 +133,15 @@ data:
     properties:
       useUnicode: true
       characterEncoding: utf8
+      # If you are running local Database instance(For example: MySQL) on the same server of minecraft.
+      # You may have the following two options uncommented.(Remove the leading "#")
+      # Since there is no need to encrypt DB connection.
+      # It helps you get rid of annoying "SSL certificate ERROR" spamming your console.
+      # More properties definitions for MySQL can be found at
+      # https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html
+      #####
```
Comment (Andre601) [inline bungee/src/main/resources/config.yml]:
```suggestion
      #
```

File: nukkit/src/main/resources/config.yml [hunk 3]
```diff
@@ -136,6 +136,15 @@ data:
     properties:
       useUnicode: true
       characterEncoding: utf8
+      # If you are running local Database instance(For example: MySQL) on the same server of minecraft.
+      # You may have the following two options uncommented.(Remove the leading "#")
+      # Since there is no need to encrypt DB connection.
+      # It helps you get rid of annoying "SSL certificate ERROR" spamming your console.
+      # More properties definitions for MySQL can be found at
+      # https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html
+      #####
```
Comment (Andre601) [inline nukkit/src/main/resources/config.yml]:
```suggestion
      #
```

File: sponge/src/main/resources/luckperms.conf [hunk 4]
```diff
@@ -136,6 +136,15 @@ data {
     properties {
       useUnicode = true
       characterEncoding = "utf8"
+      # If you are running local Database instance(For example: MySQL) on the same server of minecraft.
+      # You may have the following two options uncommented.(Remove the leading "#")
+      # Since there is no need to encrypt DB connection.
+      # It helps you get rid of annoying "SSL certificate ERROR" spamming your console.
+      # More properties definitions for MySQL can be found at
+      # https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html
+      #####
```
Comment (Andre601) [inline sponge/src/main/resources/luckperms.conf]:
```suggestion
      #
```

File: velocity/src/main/resources/config.yml [hunk 5]
```diff
@@ -126,6 +126,15 @@ data:
     properties:
       useUnicode: true
       characterEncoding: utf8
+      # If you are running local Database instance(For example: MySQL) on the same server of minecraft.
+      # You may have the following two options uncommented.(Remove the leading "#")
+      # Since there is no need to encrypt DB connection.
+      # It helps you get rid of annoying "SSL certificate ERROR" spamming your console.
+      # More properties definitions for MySQL can be found at
+      # https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html
+      #####
```
Comment (Andre601) [inline velocity/src/main/resources/config.yml]:
```suggestion
      #
```

**Comment by povsister** [comment]
Comment style fixed.
Thanks for the advice.

**Comment by lucko** [comment]
Thanks for the PR, I've changed the wording slightly and merged :)

**Pull request merged**